UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -147,6 +147,19 @@ public float MaxHealth
 		/// </summary>
 		[NonSerialized] public FireStackEvent OnClientFireStacksChange = new FireStackEvent();
 
+		// FireStacks note: It's called "stacks" but it's really just a floating point value that
+		// can go up or down based on possible sources of being on fire. Max seems to be 20 in tg.
+		private float bleedStacks => healthStateController.BleedStacks;
+
+		/// <summary>
+		/// How badly we're bleeding, same as tg bleed_stacks. 0 = not bleeding.
+		/// Exists client side - synced with server.
+		/// </summary>
+		public float BleedStacks => bleedStacks;
```

Comment:
```suggestion
		/// <summary>
		/// How badly we're bleeding, same as tg bleed_stacks. 0 = not bleeding.
		/// Exists client side - synced with server.
		/// </summary>
		// FireStacks note: It's called "stacks" but it's really just a floating point value that
		// can go up or down based on possible sources of being on fire. Max seems to be 20 in tg.
		public float BleedStacks => healthStateController.BleedStacks;
```

You'll need to replace all the `bleedStacks` with `BleedStacks` in this script